packages/shell-api/src/shell-bson.spec.ts

```diff
@@ -150,6 +150,13 @@ describe('Shell BSON', function () {
       expect(s._bsontype).to.equal('ObjectId');
       expect(s.toHexString().slice(0, 8)).to.equal('12345678');
     });
+    it('can be created using createFromHexString', function () {
+      const s = (shellBson.ObjectId as any).createFromHexString(
```

Comment:
Curious, do you know why this is not part of the exported `ObjectId` type?

Reply:
Now that you mention it, it is really exposed, so I can remove the casting to any and let it be typesafe. Just missed it because I didn't see the method in the autocomplete.

I'll push a fix to the test.

Thanks a lot!